Source Files/Writer.cpp

```diff
@@ -3,19 +3,16 @@
 #include <string>
 #include "Writer.h"

```

Comment:
For including header files convention is to include user-defined header files first and then the standard cpp library headers for readability.